definintion-of-iss-project.md

```diff
@@ -0,0 +1,58 @@
+# Title
+
+The Definition of an InnerSource Software (ISS) Project. We will largely take the ideas that follow
+from the definition of an Open Source Software (OSS) Project. Note, the definition of an OSS Project
+follows from [opensource.com's definition](https://opensource.com/resources/what-open-source). Further,
+[RedHat also supplies a defintion of OSS project](https://www.redhat.com/en/topics/open-source/what-is-open-source).
+
+Our goal here is to mimic the [open source way that opensource.com](https://opensource.com/open-source-way)
+provides. In doing this, we foster collaboration and community inside our enterprises.
+
+## Definition(s)
+
+The following are the components are essential for a project to be declared an InnerSource Software 
+project. Note, the distinction between Open Source and InnerSource is merely
+the network topology surrounding the source control management system instance. If the world at large
+can view the code base, the project is Open Source. On the other hand if any portion of the project's source 
+code lives on a restricted network topology such that it is not available to be read by the world at 
+large then we deem it InnerSource, and refer to it as "private." Also, most OSS projects contain a 
+[license (list taken from opensource.com)](https://opensource.com/law/13/1/which-open-source-software-license-should-i-use).
+The licenses are legal parameters that developers and users adhere to for the development and consumption
+of the project. Luckily we need not worry about licensing because we are concerned with "private" repositories.
```

Comment:
While I agree that the licensing issue is less of a problem with InnerSource, there are problems with large corporate constructs and taxation issues that led to the evolution of a license. Maybe it would be worthwhile to have a word with the people involved with #147 (or on the Slack, @gruetter and @danese might know more too.)